packages/components/TechSection.tsx

```diff
@@ -46,14 +46,15 @@ function TechSection() {
 
   return (
     <Section>
-      <Heading my="4rem" size="md" fontWeight={600} textAlign="left">
-        {t("tech-title")}
-      </Heading>
-      <HStack spacing="1rem" flexWrap="wrap" justifyContent="center" alignItems="center" lineHeight="200%">
-        {iconList.map((Icon) => (
-          <Icon p="0.3rem" key={Icon.name} />
-        ))}
-      </HStack>
+      <h2 className="mx-4 my-16 font-semibold text-left text-xl leading-tight">{t("tech-title")}</h2>
+
+      <div className="flex flex-wrap mx-4 items-center justify-center">
+        {iconList.map((Icon, index) => {
+          const iconSpacing = classes(index > 0 && "ml-4");
+
+          return <Icon className={iconSpacing} style={{ padding: "0.3rem" }} key={Icon.name} />;
```

Comment:
we must avoid inline styles `style={...}`.
Let's rewrite this moving the padding to some similar sizing
```suggestion
          const iconSpacing = classes("p-1", index > 0 && "ml-4");

          return <Icon className={iconSpacing} key={Icon.name} />;
```

Reply:
p-1 = `padding: 0.25rem`
p-2 = `padding: 0.5rem`